src/index.html

```diff
@@ -9,6 +9,81 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header-moyo">
+      <a class="logo-moyo" href="#home">
+        <img src="images/logo.png" alt="moyo-logo">
+      </a>
+      <nav class="nav-bar">
+        <ul class="nav__list">
+          <li class="link-item">
```

Comment:
```suggestion
      <nav class="nav">
        <ul class="nav__list">
          <li class="nav__item">
```
Якщо писати по БЕМу, то назви класів мають бути такі